pages/api.js

```diff
@@ -85,40 +87,53 @@ const API = ({ api, services }) => {
         rate_limiting={rate_limiting_resume}
       />
 
-      <Menu detail={detail} />
-
       <div id="description" className="ui container">
-        <Content content={content} />
-
-        <Access
-          access_open={access_open}
-          access_link={access_link}
-          access_description={access_description}
-          contract={contract}
-          clients={clients}
-        />
-
-        <Support
-          contact_description={contact_description}
-          contact_link={contact_link}
-        />
-
-        <Monitoring
-          monitoring_description={monitoring_description}
-          monitoring_link={monitoring_link}
-        />
-
-        <RateLimiting rate_limiting_description={rate_limiting_description} />
-
-        <Partners partners={partners} />
-
-        <TechnicalDocumentation
-          doc_tech_description={doc_tech_description}
-          doc_tech_link={doc_tech_link}
-          doc_tech_external={doc_tech_external}
-        />
-
-        <Services services={services} />
+        <ScrollableMenu detail={detail}>
+          <ScrollableMenuContext.Consumer>
```

Comment:
the page body should not appear has a children of the menu : 

- it is semantically incorrect
- it is a pitfall for rendering performance